lib/cli-engine.js

```diff
@@ -97,7 +98,9 @@ var defaultOptions = {
         cacheFile: ".eslintcache",
         fix: false,
         allowInlineConfig: true,
-        cwd: process.cwd()
+        warnIgnoredByDefault: true,
+        cwd: process.cwd(),
+        onOsCompatibilityIssue: logging.info
```

Comment:
I'd prefer not to change this interface - I don't see any good reason to expose these two new options.